docs/content/en/docs/getting-started/try-tetragon-linux.md

```diff
@@ -0,0 +1,406 @@
+using cat with:
+```shell
+cat tracing_policy.yaml
+```
+
+The output from the `tetra` CLI should be similar to:
+
+```
+🚀 process  /usr/bin/cat tracing_policy.yaml
+⁉️ syscall  /usr/bin/cat __x64_sys_openat
+⁉️ syscall  /usr/bin/cat __x64_sys_openat
+⁉️ syscall  /usr/bin/cat __x64_sys_openat
+💥 exit     /usr/bin/cat tracing_policy.yaml 0
+```
+
+{{< note >}}
+The syscall we choose is `openat` and not `open` because the glibc and most of
+the implementations of the standard C library use the `openat` syscall for
+their `open` wrapper. So `cat` on Linux is using `openat`.
+{{< /note >}}
+
+Let's dive a little deeper by inspecting the full JSON event emitted by
+Tetragon. We will use `jq`, a common command-line JSON processor, directly from
+the Tetragon container so that you don't need to install it locally.
+
+```shell
+docker exec -it tetracontainer bash
+```
+
+This should open an interactive bash session inside of the Tetragon container.
+First install `jq` and then filter on the `tetra` getevents command to select
+only the interesting events.
+
+```shell
+apk add jq
+tetra getevents | jq 'select(.process_kprobe.args[1].string_arg == "tracing_policy.yaml")
+```
+
+Again, in another terminal session, type `cat tracing_policy.yaml`. The output
+from the `tetra` CLI should be similar to this:
+
+```json
+{
+  "process_kprobe": {
+    "process": {
+      "exec_id": "OjEwMTgzOTUyNjg1NjcwNDoyNjIzNw==",
+      "pid": 26237,
+      "uid": 1000,
+      "cwd": "/home/ubuntu",
+      "binary": "/usr/bin/cat",
+      "arguments": "tracing_policy.yaml",
+      "flags": "execve clone",
+      "start_time": "2023-04-06T18:30:03.405730761Z",
+      "auid": 1000,
+      "parent_exec_id": "OjEwMTczMDEyNTQ3MjUxMDoyNjIwOA==",
+      "refcnt": 1
+    },
+    "parent": {
+      "exec_id": "OjEwMTczMDEyNTQ3MjUxMDoyNjIwOA==",
+      "pid": 26208,
+      "uid": 1000,
+      "cwd": "/home/ubuntu",
+      "binary": "/bin/bash",
+      "flags": "execve clone",
+      "start_time": "2023-04-06T18:28:14.004347210Z",
+      "auid": 1000,
+      "parent_exec_id": "OjEwMTczMDEwMjc2NTg0NjoyNjIwNw==",
+      "refcnt": 2
+    },
+    "function_name": "__x64_sys_openat",
+    "args": [
+      {
+        "int_arg": -100
+      },
+      {
+        "string_arg": "tracing_policy.yaml"
+      },
+      {
+        "int_arg": 0
+      }
+    ],
+    "action": "KPROBE_ACTION_POST"
+  },
+  "time": "2023-04-06T18:30:03.406205829Z"
+}
+```
+
+You can see that we collected information on the process itself, with for
+example, the uid, the binary name, and the start\_time. But we also collected
+information on the parent of this process, here the binary `/bin/bash`. This is
+the same kind of information we get from the base process lifecycle sensors.
+Most importantly, we can see that we hooked the function `__x64_sys_openat`
+(for an arm64 host it would be `__arm64_sys_openat`) and we also collected the
+argument's value.
```

Comment:
```suggestion
You can see that Tetragon collected information on the process itself, with for
example, the uid, the binary name, and the start\_time, as well as on the process parent,
`/bin/bash` in this case. This is
the same kind of information we get from the base process lifecycle sensors.
Finally, we can see that we hooked the function `__x64_sys_openat`
(for an arm64 host it would be `__arm64_sys_openat`) and we also collected the
argument's value.
```